test/test_intents.yaml

```
@@ -23,7 +23,7 @@
 
 en-us:
   CreateAlarm:
-    - set an alarm:
+    - set alarm:
```

Please revert this change

```suggestion
    - set an alarm:
```